jumpapp

```diff
@@ -17,14 +17,15 @@ Options:
   -t NAME -- process window has to have NAME as the window title
   -c NAME -- find window using NAME as WM_CLASS (instead of COMMAND)
   -i NAME -- find process using NAME as the command name (instead of COMMAND)
-  -w -- only find the applications in the current workspace"
+  -w -- only find the applications in the current workspace
+  -g -- switch to the last focused window when dealing with multiple matching windows"
```

Comment:
Can we make the new sort order the default and not even have a `-g` option? Does anyone object?